docs/guide/http-token.md

```diff
@@ -1,16 +1,16 @@
-# HTTP Token
+# HTTP Canarytoken
 
-## What is an HTTP token
+## What is an HTTP Canarytoken
 
 When you create a HTTP based Canarytoken, the system gives you a URL.
 
 Anyone attempting to browse to this URL will generate an alert.
 
-Why does this matter? Once you are able to get an alert for a web-based token, or a DNS based token, you have the building blocks for squillions of possible tripwires.
+Why does this matter? Once you are able to get an alert for a web-based Canarytoken, or a DNS based Canarytoken, you have the building blocks for squillions of possible tripwires.
 
-## Creating an HTTP token
+## Creating an HTTP Canarytoken
 
-Head on over to [canarytokens.org](https://canarytokens.org/generate) and select `Web bug /URL token`:
+Head on over to [canarytokens.org](https://canarytokens.org/generate) and select `Web bug`:
```

Comment:
should this say Web bug Canarytoken since this one says Canarytoken as well? 

Kubeconfig Canarytoken

Reply:
Thank you, 
yeah we could make them look the same: 
This one 'Web Bug' quotes what a user should see in the UI
Kubeconfig Canarytoken is just an hint to point which token a user needs to check.
I'd write 'Kubeconfig' in its documentation, to align it to this one

Reply:
🎉